VarduhiGhukasyan/chapter_7_8

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include<vector>
+
+void reverse_vector(const std::vector<int>& vec, int size)
+{
+	if (size == 0)
+		return;
+
+	std::cout << vec[size-1]<<" ";
```

Comment:
This is totally good solution. 
I can think of another way as well. Every time we call std::cout<< vec.pop_back() << " "; reverse_vector(vec);
pop_back() function returns last element of the vector and removes it from the vector. 
https://www.cplusplus.com/reference/vector/vector/pop_back/